go/exoservice/exoservice.go

```diff
@@ -32,17 +32,14 @@ type ReplyChannelMapping map[string]chan *structs.Message
 // Bootstrap brings an ExoService online using environment variables and the
 // given messageHandlers
 func Bootstrap(messageHandlers MessageHandlerMapping) {
-	port, err := strconv.Atoi(os.Getenv("EXOCOM_PORT"))
-	if err != nil {
-		panic(fmt.Sprintf("Invalid port: %s", os.Getenv("EXOCOM_PORT")))
-	}
+	port := getPort()
```

Comment:
Can this variable be removed and we can `getPort()` in the definition of `config`

Reply:
I did that at first but there's a print statement on line 46 that needs this var

Reply:
Danget, I looked for that and couldn't find it. Github did not expand the diff the way I expected. Good as it